mappings/net/minecraft/world/gen/chunk/ChunkGeneratorType.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/class_5284 net/minecraft/world/gen/chunk/ChunkGeneratorType
+	FIELD field_24514 defaultBlock Lnet/minecraft/class_2680;
+	FIELD field_24515 defaultFluid Lnet/minecraft/class_2680;
+	FIELD field_24516 config Lnet/minecraft/class_5311;
+	FIELD field_24783 bedrockCeilingY I
+	FIELD field_24784 bedrockFloorY I
```

Comment:
These do not match the strings in the codec
`bedrock_roof_position` and `bedrock_floor_position`

Reply:
Position isn't as accurate as `level` or `y`. fyi this name is from before mojang exposed these names, so.